backend/Squirrel.ConsoleApp/Extensions/HubConnectionExtensions.cs

```diff
@@ -0,0 +1,220 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.Extensions.DependencyInjection;
+using Squirrel.ConsoleApp.BL.Interfaces;
+
+namespace Squirrel.Core.WebAPI.Extensions;
+
+public static class HubConnectionExtensions
+{
+    private static bool CurrentClientIsTarget(string currentClientId, string clientId)
+    {
+        if (string.IsNullOrEmpty(currentClientId))
+        {
+            return false;
+        }
+
+        if (!clientId.Equals(currentClientId))
+        {
+            return false;
+        }
+
+        return true;
+    }
```

Comment:
I think, that is not okay to notify all clients. This could lead to leaks of user information.

https://learn.microsoft.com/uk-ua/aspnet/core/signalr/groups?view=aspnetcore-6.0
We can use GUID to identify the target of notification.

Reply:
return A && B, no need for so many lines

Reply:
@Zoranais and I have solved problem with notifying all clients. For now we use Client Id to identify the target of notification. In this case we don't need additional checks anymore and I have already deleted them.